test/unit/src/site/_filters/algolia-item.js

```diff
@@ -5,18 +5,34 @@ const {
   imgixDomain,
 } = require('../../../../../src/site/_data/site');
 const algoliaItem = require('../../../../../src/site/_filters/algolia-item');
+const {memoize} = require('../../../../../src/site/_filters/find-by-url');
 
 const createPost = () =>
   /** @type {EleventyCollectionItem} */ ({
     data: {
       lang: defaultLocale,
       title: 'Hello world',
+      page: {
+        filePathStem: '/en/parent/hello-world/index',
+      },
     },
     templateContent: 'Hello world',
     url: '/hello-world/',
   });
 
+/**
+ * Adds parent to memorized posts for following tests
+ */
+const initParent = () => {
+  const mockParent = createPost();
+  mockParent.data.title = 'Parent Title';
+  mockParent.url = '/parent/';
+  memoize([mockParent]);
+};
+
```

Comment:
I have a hard time explaining what feels off with this test. Here are the bits: the `findByUrl` module has its own test suite actually, importing another module than the tested one into a test suite is not the best practice I'd say, but things should rather be mocked instead.

Mocking though is what happens above in `createPost` and as the `algolia-item` filter has barely any logic besides prop-getting it is barely testing anything.

I'd vote to extract the 

```js
const parentUrl = `${post.data.page.filePathStem
    .split('/')
    .slice(0, -2)
    .join('/')}/`;
``` 

bit to a separate function and test this instead. And simply leave this test suite here alone. Thanks for thinking about updating it though!